lGlobal_total_timesteps: 10000000, n_steps_total: 10000, lEpochs: 6, lDeltaStepsEpoch: 1666667
starting epoch 0, total_timesteps: 1666667, listTowns: ['Town01', 'Town02', 'Town03', 'Town04']
Neil run/train_rl_parent_NeilBranch0.py:104
run/train_rl_NeilBranch0.sh: line 21: /home/nsambhu/miniconda3/etc/profile.d/conda.sh: No such file or directory

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


start at Tue Sep 13 15:44:30 EDT 2022
CarlaUE4-Linux: no process found
[2022-09-13 15:44:32,998][utils.server_utils][INFO] - Kill Carla Servers!
3c80c96f15cb
[2022-09-13 15:44:33,637][utils.server_utils][INFO] - Kill Carla Servers!
[2022-09-13 15:44:33,637][utils.server_utils][INFO] - echo q | sudo -S docker run --privileged --gpus "device=0" --net=host -v /tmp/.X11-unix:/tmp/.X11-unix:rw carlasim/carla:0.9.13 /bin/bash ./CarlaUE4.sh -carla-rpc-port=2000
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:90
[sudo] password for nsambhu: sh: 1: xdg-user-dir: not found
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:100
[2022-09-13 15:44:34,530][utils.server_utils][INFO] - echo q | sudo -S docker run --privileged --gpus "device=0" --net=host -v /tmp/.X11-unix:/tmp/.X11-unix:rw carlasim/carla:0.9.13 /bin/bash ./CarlaUE4.sh -carla-rpc-port=2005
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:90
[sudo] password for nsambhu: sh: 1: xdg-user-dir: not found
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:100
[2022-09-13 15:44:35,351][utils.server_utils][INFO] - echo q | sudo -S docker run --privileged --gpus "device=0" --net=host -v /tmp/.X11-unix:/tmp/.X11-unix:rw carlasim/carla:0.9.13 /bin/bash ./CarlaUE4.sh -carla-rpc-port=2010
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:90
[sudo] password for nsambhu: sh: 1: xdg-user-dir: not found
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:100
[2022-09-13 15:44:36,173][utils.server_utils][INFO] - echo q | sudo -S docker run --privileged --gpus "device=0" --net=host -v /tmp/.X11-unix:/tmp/.X11-unix:rw carlasim/carla:0.9.13 /bin/bash ./CarlaUE4.sh -carla-rpc-port=2015
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:90
[sudo] password for nsambhu: sh: 1: xdg-user-dir: not found
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Neil /home/nsambhu/github/carla-roach/utils/server_utils.py:100
